var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -263,6 +263,9 @@ def make_defs(self):
             'FC={0}'.format(spack_fc),
         ]
 
+        if self.spec.os == 'monterey':
+            make_defs.append('MACOSX_DEPLOYMENT_TARGET=11.0')
```

Comment:
A little more explanation wouldn't go amiss 😬 
I guess the .0 is not really significant and the point is 11 (Big Sur) vs 12 (Montery)? Curious because my Big Sur machine is on 11.6...